aap_chatbot/src/useChatbot/useChatbot.ts

```diff
@@ -77,7 +79,9 @@ export const feedbackMessage = (f: ChatFeedback): MessageProps => ({
   content:
     f.sentiment === Sentiment.THUMBS_UP
       ? "Thank you for your feedback!"
-      : "Thank you for your feedback. If you have more to share, please click the button below (_requires GitHub login_).",
+      : "Thank you for your feedback. If you have more to share, please click the button below (_requires GitHub login_). " +
+        "\\n\\n Do not include any personal information or other sensitive information in your feedback. " +
+        "Feedback may be used to improve Red Hat's products or services.",
```

Comment:
>Red Hat's products or services.
Is this statement finalized, IMO this seems bit confusing as feedback should be given to improve the chatbot response/suggestion for the raised issue, and not the Red Hat's products or services.

Reply:
Yeahh... this is the concrete text supposed to be replaced by, as from: https://issues.redhat.com/browse/AAP-43500